frontend/src/editor/DeleteButton.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import { _ } from "../i18n";
+  import { modKey } from "../keyboard-shortcuts";
+
+  export let deleting: boolean;
+
+  $: buttonContent = deleting ? _("Deleting...") : _("Delete");
+</script>
+
+<button type="submit" value="Delete" title={`${_("Delete")} (${modKey}+s)`}>
```

Comment:
I'd just have this as a `type="button"` and then have a separate event handler for this one.

Reply:
Yeah I think that simplifies things quite a bit